notebook/static/notebook/js/notebook.js

```diff
@@ -2845,6 +2845,57 @@ define([
         }
     };
     
+       
+    Notebook.prototype.save_notebook_as = function() {
+            // If current notebook has some changes, save them
+            if (this.writable && this.dirty) {
+                this.save_notebook();
```

Comment:
I don't think I'd expect 'save as' to save over the old name as well, though it's a bit complicated by autosave. @minrk @gnestor @Carreau ?

Reply:
This is to save the current notebook before navigating away. We did agree that "Save as" would open the newly created notebook (Right? Did I misunderstand?), so I thought we should try to save the current notebook before moving away. I think "make a copy" and "trust" do this too.

Reply:
> This is to save the current notebook before navigating away. We did agree that "Save as" would open the newly created notebook (Right? Did I misunderstand?), so I thought we should try to save the current notebook before moving away. I think "make a copy" and "trust" do this too.

If you do open the new notebook, then this will likely be almost identical to "Make a Copy", Should we have "Make a copy" take a parameter which is the new name/path ?